modules/misc/news.nix

```diff
@@ -103,7 +103,7 @@ in
       entries = mkOption {
```

Comment:
This file should not be formatted at this time. Use the `format` script in the project root to format the files instead, it will automatically filter out this and a few more files.